plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClientModule.java

```diff
@@ -78,6 +81,13 @@ public static ConnectionFactory connectionFactory(BaseJdbcConfig config, Credent
                 new OracleDriver(),
                 config.getConnectionUrl(),
                 connectionProperties,
-                credentialProvider));
+                credentialProvider),
+                OracleClientModule::isRetryableForOpeningConnection);
```

Comment:
Oracle connector always uses `RetryingConnectionFactory` regardless of the configuration for backward compatibility.